options/path.c

```diff
@@ -217,7 +217,8 @@ char *mp_get_user_path(void *talloc_ctx, struct mpv_global *global,
     }
     if (!res)
         res = talloc_strdup(talloc_ctx, path);
-    MP_DBG(global, "user path: '%s' -> '%s'\n", path, res);
+    int msg_level =  strcmp(path, res) ? MSGL_DEBUG : MSGL_TRACE;
```

Comment:
oh, I don't like `strcmp` here, it is performance penalty for long data inputs. Shouldn't we just `mp_is_url` and return if url? I guess it covers all protocols and so on.

Reply:
Actually I can just use `!res` to accomplish this duh.

Reply:
But still, I think it doesn't make sense to run all this code on `mp_is_url` paths.

Reply:
Is it so much code? Any `mp_is_url` string should instantly fail the `bstr_eatstart0` part. Or do you just mean not log at all.

Reply:
Oh, you are right.

> Or do you just mean not log at all.

In fact I like it. Put the log into `if (bstr_eatstart0(&bpath, "~"))`, else we just are transparent.

and... I still don't like this `talloc_strdup` when we are not expanding. But I guess it;s needed.

Reply:
Yeah I'm thinking the trace log is useless info on further thought. It should just be only if the path expands.